pyanaconda/modules/localization/localization_interface.py

```diff
@@ -68,6 +69,15 @@ def GetLocales(self, language_id: Str) -> List[Str]:
         """
         return self.implementation.get_locales(language_id)
 
+    def GetCommonLocales(self) -> List[Str]:
+        """Get a list of the most commonly used locales.
+
+        For example: ["ar_EG.UTF-8", "en_US.UTF-8", "en_GB.UTF-8", ...]
+
+        :return: a list of common locale IDs.
+        """
+        return langtable.list_common_locales()
```

Comment:
We follow the Proxy software design pattern here (https://en.wikipedia.org/wiki/Proxy_pattern). The `*Interface` classes are just proxies between the actual implementation and a message bus. Their only job is to convert the values into something that can be send on DBus and back.

In this case, you should call `self.implementation.get_common_locales()` and use `langtable` in the `LocalizationService` class.